src/core/qgsimagecache.cpp

```diff
@@ -66,7 +67,7 @@ int QgsImageCacheEntry::dataSize() const
   int size = 0;
   if ( !image.isNull() )
   {
-    size += ( image.width() * image.height() * 32 );
+    size += ( image.width() * image.height() * 4 );
```

Comment:
```suggestion
    size += image.sizeInBytes();
```

It's more accurate, since it'll correctly handle different colour depths.

Reply:
Done